integration/testdata/fluentd-multiple-lockfiles.cdx.json.golden

```diff
@@ -121,7 +121,12 @@
       "licenses": [
         {
           "license": {
-            "name": "GPL-2.0"
+            "name": "GPL-2.0-or-later"
+          }
+        },
+        {
+          "license": {
+            "name": "GPL-2.0-only"
```

Comment:
I didn't find why we use `*-only` and `*-or-later` licenses here

Reply:
This is explained in https://github.com/aquasecurity/trivy/blob/85b878105b23c8c03b7721667f1fe92fb438a2de/pkg/fanal/analyzer/pkg/dpkg/copyright_test.go#L69 where the `apt` copyright has both:

- License: GPLv2+ interpreted as GPL-2.0-or-later
- /usr/share/common-licenses/GPL-2 interpreted as GPL-2.0-only